bindings/py/cpp_src/bindings/algorithms/py_Cells4.cpp

```diff
@@ -283,7 +283,7 @@ namespace nupic_ext
         py_cells4.def("getCell", &Cells4_t::getCell);
         py_cells4.def("getCellIdx", &Cells4_t::getCellIdx);
 
-        py_cells4.def("getSegment", &Cells4_t::getSegment, py::return_value_policy::reference);
+//        py_cells4.def("getSegment", &Cells4_t::getSegment, py::return_value_policy::reference);
```

Comment:
forgotten test? Should stay valid

Reply:
Hmmm, thought I got rid of that...but then I did a merge after your changes.